analytic_multicurrency/tests/test_analytic_lines.py

```diff
@@ -0,0 +1,489 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Damien Crier
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import time
+from openerp.tests import common
+
+
+class testAnalyticLine(common.TransactionCase):
+
+    def setUp(self):
+        super(testAnalyticLine, self).setUp()
+        self.agrolait = self.ref('account.analytic_agrolait')
```

Comment:
If those are new tests, it would be great to have them in new API. That means:
- `self.ref` -> `self.env.ref` (it returns a `browse_record`, noo need to `browse()`)
- `self.registry` -> `self.env`
- no need for `model_data_obj.get_object_reference`: you can use `self.env.ref`

Thanks!